src/ows/src/test/java/org/geoserver/ows/OWS10ServiceExceptionHandlerTest.java

```diff
@@ -187,4 +218,17 @@ public void testHandleServiceExceptionNullMessages() throws Exception {
         // used to contain an extra " null" at the end
         assertEquals("hello service exception NullPointerException", exceptionText);
     }
+
+    @Test
+    public void exceptionType() throws Exception {
```

Comment:
Do you think a similar test is needed also for OWS 1.1?

Reply:
@mbarto Added a test for that too.